src/cargo/ops/cargo_output_metadata.rs

```diff
@@ -74,7 +74,6 @@ impl Encodable for MetadataResolve {
     fn encode<S: Encoder>(&self, s: &mut S) -> Result<(), S::Error> {
         #[derive(RustcEncodable)]
         struct EncodableResolve<'a> {
-            root: &'a PackageId,
```

Comment:
Hm, actually there is absolutely no need to break anything untill cargo metadata supports workspaces. Will rebase in a moment.